### ISSUE TITLE
Make a separate downloadable module for neoplastic/abnormal cells

### ISSUE THREAD

**Issue by cmungall** [NCI-Thesaurus/thesaurus-obo-edition#31]
It would be useful to have a module `ncit/subsets/abnormal-cell.owl`

This would contain the hierarchy under:

[NCIT:C12913](http://purl.obolibrary.org/obo/NCIT_C12913) ! Abnormal Cell

E.g.

[NCIT:C48361](http://purl.obolibrary.org/obo/NCIT_C48361) ! Neoplastic Adrenal Cortical Cell

Note that there is currently no ontology of abnormal cell types in OBO. I think this should become it.

We would want to wire the NCIT classes to CL classes. It looks like the abnormal cell hierarchy is under-axiomatized in NCIT at the moment. We could work on this together and come up with some simple DPs for the most obvious ones.

E.g.

"NCIT:Abnormal <X> cell" = "NCIT:Abnormal cell" and variant-of some "CL:<X> cell"

**Comment by NCIEVS** [comment]
Sounds like a good idea.  There are no restrictions on abnormal cell classes afaik. 

We publish the isolated hierarchies (and a couple of sub-branches) in our ftp site.  The abnormal cell tree is in ftp://ftp1.nci.nih.gov/pub/cacore/EVS/NCI_Thesaurus/Branches/Abnormal_Cell.zip.  But I gather you'd be generating that subset after the OBO-ization step?

**Comment by cmungall** [comment]
> you'd be generating that subset after the OBO-ization step?

Yes, that would be the idea

**Comment by cmungall** [comment]
> As far as I know, the axioms for the abnormal cell types should read something like:
> abnormal cell type X derives from some CL:cell type Y

Is this always true? Some cells are born abnormal, and may be derived from preserved specification and determination pathways from a different abnormal cell type.

So the lineage may be `A -> B -> AbB -> AbC`, where `AbC` is the abnormal form of `C`. 

> Alex Diehl is working on a cancer cell ontology. We should coordinate with him.

@addiehl - can you post details?

**Comment by wdduncan** [comment]
I won't lie ... I don't understand biology well enough to answer your question. Do you have a specific example?

Follow up questions:
1. Does the develops_from relation address your example?
2. I thought the derives_from and develops_from relations "hide" a number of details regarding lineage and mitosis. Yes? No? If so, then perhaps your example is making this clear.
3. Does an axiom like AbC derives_from AbB and derives_from C address your example?

**Comment by cmungall** [comment]
You can look at my example abstractly, regardless of details of mitosis and specific relation.

The point is that it would be wrong to say every 'abnormal-C' derives from a normal C, if some abnormal-Cs have never been a C, they haven't been normal since they were a B or A.

**Comment by wdduncan** [comment]
Ok. What do you propose?

**Comment by wdduncan** [comment]
Yes, similarity is vague. Do you know any cancer pathologists? How similar does a malignant cell's phenotype need be a normal cell's phenotype in order for the determination to be made that the malignant cell originated from certain normal cell type? I am imagine there is a lot of variance.
